contract/src/main/kotlin/io/provenance/scope/loan/utility/DataExtensions.kt

```diff
@@ -0,0 +1,22 @@
+package io.provenance.scope.loan.utility
+
```

Comment:
Aside from resolving errors of undefined extension functions, I thought it would be ideal to have a file like this where you can clearly see the kind of data validation being done for specific data types.